src/core/qgsapplication.h

```diff
@@ -465,6 +471,7 @@ class CORE_EXPORT QgsApplication : public QApplication
     QgsActionScopeRegistry* mActionScopeRegistry;
     QgsRuntimeProfiler* mProfiler;
     QgsTaskManager* mTaskManager;
+    QgsFieldFormatterRegistry* mFieldFormatterRegistry;
```

Comment:
I don't have a strong preference here but I think this should be discussed separate from this PR.